--- Varien.php	2016-10-04 00:01:09.164681747 +1100
+++ Varien.php.new	2016-10-03 23:59:55.965983004 +1100
@@ -101,19 +101,19 @@
             'httponly' => $cookie->getHttponly()
         );
 
-        if (!$cookieParams['httponly']) {
-            unset($cookieParams['httponly']);
-            if (!$cookieParams['secure']) {
-                unset($cookieParams['secure']);
-                if (!$cookieParams['domain']) {
-                    unset($cookieParams['domain']);
-                }
-            }
-        }
+        #if (!$cookieParams['httponly']) {
+        #    unset($cookieParams['httponly']);
+        #    if (!$cookieParams['secure']) {
+        #        unset($cookieParams['secure']);
+        #        if (!$cookieParams['domain']) {
+        #            unset($cookieParams['domain']);
+        #        }
+        #    }
+        #}
 
-        if (isset($cookieParams['domain'])) {
-            $cookieParams['domain'] = $cookie->getDomain();
-        }
+        #if (isset($cookieParams['domain'])) {
+        #    $cookieParams['domain'] = $cookie->getDomain();
+        #}
 
         call_user_func_array('session_set_cookie_params', $cookieParams);
 
